Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/07-SyncUpDetailNavigation/MeetingNavigation-01-code-0003.swift

```diff
@@ -23,6 +23,31 @@ struct MeetingView: View {
   }
 }
 
+extension SyncUp {
```

Comment:
Adding `SyncUp.mock` while we're making the `MeetingView`, which gets used later on in the tutorial